atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -462,7 +464,7 @@ public String toString() {
     @Override
     public Map<Cell, Value> get(TableReference tableRef, Map<Cell, Long> timestampByCell) {
         if (timestampByCell.isEmpty()) {
-            log.info("Attempted get on '{}' table with empty cells", tableRef);
+            log.info("Attempted get on '{}' table with empty cells", UnsafeArg.of("tableRef", tableRef));
```

Comment:
Table names are dynamically created by certain AtlasDB clients and could contain sensitive information..